docs/howto-include_images_in_exercises.md

```diff
@@ -0,0 +1,19 @@
+How to include static content in an Exercise Statement
+======================================================
```

Comment:
Maybe this should be rephrased to match the filename/url ?
```suggestion
How to include images in exercises
==================================
```

BTW, please rename the .md so that it only has dashes:

```
git mv docs/howto-include_images_in_exercises.md docs/howto-include-images-in-exercises.md
git commit -m "Rename"
```